From 6f9e2a433e1c7368295e940a8af91101ec225966 Mon Sep 17 00:00:00 2001
From: Sergey Temerkhanov <s.temerkhanov@gmail.com>
Date: Thu, 5 May 2016 19:27:44 -0700
Subject: [PATCH 0403/1239] cavium: Drop non-OF code

Non-OF code has been obsoleted for some time. Drop it.

Signed-off-by: Sergey Temerkhanov <s.temerkhanov@gmail.com>
---
 board/cavium/thunderx/thunderx.c | 28 +++-------------------------
 1 file changed, 3 insertions(+), 25 deletions(-)

diff --git a/board/cavium/thunderx/thunderx.c b/board/cavium/thunderx/thunderx.c
index 35bb72854b..67f6c2015a 100644
--- a/board/cavium/thunderx/thunderx.c
+++ b/board/cavium/thunderx/thunderx.c
@@ -12,14 +12,11 @@
 
 #include <linux/compiler.h>
 
-#include "cavm-csr.h"
 #include <cavium/atf.h>
 
-#ifdef CONFIG_OF_LIBFDT
- #include <libfdt.h>
- #include <fdt_support.h>
- #include <cavium/thunderx_fdt.h>
-#endif
+#include <libfdt.h>
+#include <fdt_support.h>
+#include <cavium/thunderx_fdt.h>
 #include <cavium/atf.h>
 #include <asm/armv8/mmu.h>
 
@@ -62,10 +59,8 @@ DECLARE_GLOBAL_DATA_PTR;
 
 int board_init(void)
 {
-#ifdef CONFIG_OF_LIBFDT
 	ulong fdt_addr = (ulong)gd->fdt_blob;
 	set_working_fdt_addr(fdt_addr);
-#endif
 
 	return 0;
 }
@@ -116,25 +111,8 @@ void reset_cpu(ulong addr)
  */
 int board_late_init(void)
 {
-#ifdef CONFIG_OF_LIBFDT
 	thunderx_parse_bdk_config();
-#else
-	int i;
-	char str[32];
-	const char *boardname;
-
-	for (i = 0; i < atf_env_count(); i++) {
-		atf_env_string(i, str);
 
-		debug("Environment string %d: %s\n", i, str);
-
-		if (!strncmp(str, BOARD_TYPE, strlen(BOARD_TYPE))) {
-			boardname = str + strlen(BOARD_TYPE);
-			setenv("board", boardname);
-			break;
-		}
-	}
-#endif
 	printf("Board type: %s\n", getenv("board"));
 
 	return 0;
-- 
2.29.0

